README.md

```diff
@@ -1,15 +1,15 @@
 [![CircleCI](https://circleci.com/gh/ministryofjustice/community-api/tree/main.svg?style=svg)](https://circleci.com/gh/ministryofjustice/community-api)
 [![Docker Repository on Quay](https://quay.io/repository/hmpps/community-api/status "Docker Repository on Quay")](https://quay.io/repository/hmpps/community-api)
 [![Runbook](https://img.shields.io/badge/runbook-view-172B4D.svg?logo=confluence)](https://dsdmoj.atlassian.net/wiki/spaces/NOM/pages/1739325587/DPS+Runbook#Deploying-Community-API)
-[![API docs](https://img.shields.io/badge/API_docs_-view-85EA2D.svg?logo=swagger)](https://community-api-public.test.delius.probation.hmpps.dsd.io/swagger-ui/index.html)
+[![API docs](https://img.shields.io/badge/API_docs_-view-85EA2D.svg?logo=swagger)](https://community-api.test.probation.service.justice.gov.uk/swagger-ui/index.html)
 
 # Community API
 Spring Boot 2, Java 11 API for accessing Probation offenders information
 
 The service provides REST access to the Delius Oracle DB.
 
 Documentation including URLs for the various endpoints can be found on the test instance at
-https://community-api-public.test.delius.probation.hmpps.dsd.io/swagger-ui/index.html (VPN / trusted network required).
+https://community-api.test.probation.service.justice.gov.uk/swagger-ui/ (VPN / trusted network required).
```

Comment:
slightly odd that you've taken off `index.html` here, but is on the url earlier.  I think both work anyway...

Reply:
For info, the https://community-api-public.test.probation.service.justice.gov.uk url redirects to /swagger-ui/index.html anyway - so you could leave it off if you wanted.